### ISSUE TITLE
[Feat] Improve APIs of the Spark library to be more concise for easily importing/exporting/transforming data at the graph level

### ISSUE THREAD

**Issue by acezen** [alibaba/GraphAr#88]
**Is your feature request related to a problem? Please describe.**
as title

**Describe the solution you'd like**
Proposal TBF

**Comment by acezen** [comment]
### Load GAR file format as DataFrames
```scala
object GraphReader {
     /*
      * Input:  graph yaml path
      * Output: vertex DataFrames and edge DataFrames
      * /
     def read(graph_yaml_path): Pair[Map[String, DataFrame], Map[String, DataFrame]] {}
     
     /*
      * Input:  graph info
      * Output: vertex DataFrames and edge DataFrames
      * /
     def read(graph_info): Pair[Map[String, DataFrame], Map[String, DataFrame]] {}
}

### Write a set of DataFrames to GAR files
```scala
object GraphWriter {
     /*
      * Input:  graph yaml path and a set of DataFrames
      * Status:  success or fail
      * /
     def write(graph_yaml_path,  vertexDataFrames: Map[String, DataFrame], edgeDataFrame: Map[String, DataFrame]): Status {}
     
     /*
      * Input:  graph info and a set of DataFrames
      * Status:  success or fail
      * /
     def write(graph_info, vertexDataFrames: Map[String, DataFrame], edgeDataFrame: Map[String, DataFrame]): Status {}
}

### Transform graph from one type of GAR to another type of GAR
```scala
object GraphTransFormaer {
     /*
      * Input:  source and dest graph yaml path
      * Status:  success or fail
      * /
     def transform(source_graph_yaml_path, dest_graph_yaml_path): Status {}
     
     /*
      * Input:  source and dest graph info
      * Status:  success or fail
      * /
     def transform(source_graph_info, dest_graph_info): Status {}
}

**Comment by acezen** [comment]
Solved by pull requests #113 #114
